src/pages/ReimbursementAccount/BusinessInfo/BusinessInfo.tsx

```diff
@@ -67,19 +40,27 @@ const bodyContent = [
 
 const businessInfoStepKeys = CONST.BANK_ACCOUNT.BUSINESS_INFO_STEP.INPUT_KEY;
 
-function BusinessInfo({reimbursementAccount, reimbursementAccountDraft, policyID, onBackButtonPress, onCloseButtonPress}) {
+type BusinessInfoOnyxProps = {
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+    reimbursementAccountDraft: OnyxEntry<ReimbursementAccountDraft>;
+};
+
+type BusinessInfoProps = {
+    reimbursementAccount: ReimbursementAccount;
+    reimbursementAccountDraft: ReimbursementAccountDraft;
+    policyID: string;
```

Comment:
Should it be required? I see it was optional before. If it's an optional get back the default value for it

Reply:
it was optional as per prop types but it doesn't really make sense when looking at `updateCompanyInformationForBankAccount`, which requires it